internal/action/bufpane.go

```diff
@@ -570,7 +568,7 @@ func (h *BufPane) execAction(action BufAction, name string, te *tcell.EventMouse
 	case BufMouseAction:
 		success = a(h, te)
 	}
-	success = success && h.PluginCB("on"+name)
+	success = success && h.PluginCB("on"+name, te)
```

Comment:
We add this now also for `BufKeyAction` too, right?
Furthermore...is this optional argument really backward compatible? :thinking:

Reply:
> We add this now also for `BufKeyAction` too, right?

Yes, but it is nil in that case.



> Furthermore...is this optional argument really backward compatible? 🤔

Non-intuitively yes. You can try it. A lua function is free to receive less arguments than we pass to it, or more arguments than we pass to it.